src/main/java/backend/Season.java

```diff
@@ -1,8 +1,9 @@
 package backend;
 
 import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
 
```

Comment:
I still prefer this grouping of imports. It is the default, enforced by IntelliJ, and also in terms of logic it makes sense to separate the "core" imports from the gson imports.